lenovo/ideapad/14imh9/default.nix

```diff
@@ -0,0 +1,122 @@
+{ lib, pkgs, ... }:
+
+{
+  imports = [
+    ../../../common/cpu/intel/meteor-lake
+    ../../../common/pc/laptop
+    ../../../common/pc/ssd
+  ];
+
+  boot = {
+    # Workaround: Out of the box, resuming from hibernation will break sounds.
+    #             See https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/issues/766
+    extraModprobeConfig = ''
+      options snd-hda-intel model=generic
+      options snd-hda-intel snd-intel-dspcfg.dsp_driver=1
+      blacklist snd_soc_skl
+    '';
+
+    initrd.availableKernelModules = [
+      # Necessary mods
+      "xhci_pci"
+      "nvme"
+      "usbhid"
+      "usb_storage"
```

Comment:
We actually have nvme enabled by default in nixos. In general we are trying not to replicate nixos-generate-config and complement it. So unless there is some module that doesn't get detected by nixos-generate-config please remove it.